pages/login.page.js

```diff
@@ -58,6 +59,22 @@ class LoginPage extends WebPage {
         })();  
     }
 
+    async alertSignupIsVisible() {
+        let ec = await browser.ExpectedConditions;
+        await browser.wait(ec.visibilityOf($('#create_account_error>ol>li')), 4000); 
```

Comment:
Create `waitForVisible()` method in the base element, using the same approach.
After, here, you can simply use `this.getAlertSignupBlock().waitForVisible()`

Reply:
Done